spec/lcm/integration/shared_examples_for_synchronization_bricks.rb

```diff
@@ -111,19 +110,14 @@
   end
 
   it 'migrates reports used in dashboards' do
-    original_reports = original_project.reports.to_a
-    used_reports_titles = ['Customers Count By State', 'Customers Count By Gender', 'Report contains CA']
-    expected_reports = original_reports.find_all do |r|
-      used_reports_titles.include?(r.title)
-    end
-    expect(expected_reports.length).to be 3
+    used_reports = original_project.dashboards.map{|d|d.tabs.map{|t|t.items.map{|i|i}}}.flatten.map{|r|r.obj}.uniq{|r|r.identifier}
```

Comment:
Identical code found in shared_examples_for_synchronization_bricks.rb:113, shared_examples_for_synchronization_bricks.rb:129 (mass = 112)